README.md

```diff
@@ -183,10 +183,12 @@ $builder->info($text, $title = null);
 ```php
 $builder->listCheck(array $list, $text = null, $selected = null);
```

Comment:
`$selected` should be an array here.

Can you amend your change to explain why? Thanks!